rps.js

```diff
@@ -0,0 +1,32 @@
+let choice = prompt("Please choose one from 'rock','paper','scissors'"); 
```

Comment:
Since the question asked for a function every thing under this line should be in a function and take a `choice` as parameter.
so your code should look like this like this 
```
let choice = prompt("Please choose one from 'rock','paper','scissors'"); 
const play = function(choice){
     // The rest of your code here
}
// Here call the function and give it the choice variable as an argument 
play(choice);
```